### PR TITLE
Add geo polygon GSoC article

### PR DESCRIPTION
**Pull request by zzzz-vincent**
This PR should be merged after #316

### CODE REVIEW

File: qdrant-landing/content/articles/geo-polygon-filter-gsoc.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    - geo polygon
+    - search condition
+    - gsoc'23 
+---
+
+
+
+## Introduction
+
+Greetings, I'm Zein Wen, and I was a Google Summer of Code 2023 participant at Qdrant. I got to work with an amazing mentor, Arnaud Groulay, on enhancing the Qdrant Geo Polygon Filter.. This new feature allows users to refine their query results using polygons. As the latest addition to the Geo Filter family of radius and rectangle filters, this enhancement promises greater flexibility in querying geo data, unlocking interesting new use cases.
```
Comment (agourlay) [inline qdrant-landing/content/articles/geo-polygon-filter-gsoc.md:25]:
```suggestion
Greetings, I'm Zein Wen, and I was a Google Summer of Code 2023 participant at Qdrant. I got to work with an amazing mentor, Arnaud Gourlay, on enhancing the Qdrant Geo Polygon Filter.. This new feature allows users to refine their query results using polygons. As the latest addition to the Geo Filter family of radius and rectangle filters, this enhancement promises greater flexibility in querying geo data, unlocking interesting new use cases.
```

:stuck_out_tongue_closed_eyes:

Comment (agourlay) [inline qdrant-landing/content/articles/geo-polygon-filter-gsoc.md:25]:
```suggestion
Greetings, I'm Zein Wen, and I was a Google Summer of Code 2023 participant at Qdrant. I got to work with an amazing mentor, Arnaud Groulay, on enhancing the Qdrant Geo Polygon Filter. This new feature allows users to refine their query results using polygons. As the latest addition to the Geo Filter family of radius and rectangle filters, this enhancement promises greater flexibility in querying geo data, unlocking interesting new use cases.
```

**Comment by agourlay** [comment]
The article does not appear in the right position in https://deploy-preview-333--condescending-goldwasser-91acf0.netlify.app/articles/#

blog-order

**Comment by kacperlukawski** [comment]
How about putting the following image as a social preview?

https://cdn.discordapp.com/attachments/1143906349663649822/1161200303945830460/kacperlukawski_Android_with_a_map_in_the_background_floating_co_6f3b9f4f-7d97-4482-9e36-0f2fb143b880.png?ex=65376f0e&is=6524fa0e&hm=67e4f720dee78a48ebf7fa03a3130428e0644e36220ca00d633de9c2cb1e77e4&

**Comment by zzzz-vincent** [comment]
Change was pushed :)

**Review by agourlay — APPROVED** [review]

**Pull request merged**